twitch/api/users.py

```diff
@@ -26,9 +27,25 @@ def check_subscribed_to_channel(self, user_id, channel_id):
         response = self._request_get('users/{}/subscriptions/{}'.format(user_id, channel_id))
         return Subscription.construct_from(response)
 
+    def _get_all_follows(self, user_id, direction, sort_by):
+        offset = 0
+        params = {
+            'limit': MAX_FOLLOWS_LIMIT,
+            'direction': direction
+        }
+        follows = []
+        while offset is not None:
+            params.update({
+                'offset': offset
+            })
+            response = self._request_get('users/{}/follows/channels'.format(user_id), params=params)
+            offset = response.get('_offset')
+            follows.extend(response['follows'])
+        return [Follow.construct_from(x) for x in follows]
+
     def get_follows(self, user_id, limit=25, offset=0, direction=DIRECTION_DESC,
-                    sort_by=USERS_SORT_BY_CREATED_AT):
-        if limit > 100:
+                    sort_by=USERS_SORT_BY_CREATED_AT, get_all=False):
```

Comment:
Rather than changing a `get_follows` function and adding a parameter that changes how the function behaves, could you create another function `get_all_follows` or something like that, that would return all follows by a user?
It's much nicer to have two functions than one being changed by a parameter.